### DISCUSSION TITLE
reassemble equivalent in Get

### DISCUSSION THREAD

**Discussion by mrnusa** [jonataslaw/getx#1112] [Q&A]
Hi, 

what is the reassemble(https://api.flutter.dev/flutter/widgets/State/reassemble.html) equivalent in Get? How can i track hot reload?

thanks

**Comment by eduardoflorence** [comment]
```dart
Get.forceAppUpdate()
```

**Reply by mrnusa** [reply]
okay, but it is a function to force redraw widgets. But i need to override the reassemble method to call an another command.
I need for that package:
https://pub.dev/packages/qr_code_scanner/example

this is the part of code what i would like to done with Get:
```
  @override
  void reassemble() {
    super.reassemble();
    if (Platform.isAndroid) {
      controller.pauseCamera();
    }
    controller.resumeCamera();
  }
```

**Reply by enosads** [reply]
@mrnusa have you find a solution? @mrnusa

**Reply by mrnusa** [reply]
> @mrnusa have you find a solution? @mrnusa

no :(

**Reply by Gopi-p** [reply]
Have you find a way to achieve this? @mrnusa

**Reply by moshTech** [reply]
You can be using StatefulWidget which gives you reassemble() while debugging then switch to getx when you want to release. It is only needed in hot reload.

**Reply by Gopi-p** [reply]
Ok thanks :)
